tools/gource/gource.sh

```diff
@@ -19,7 +19,7 @@ gource                          \
   --max-file-lag 0.05           \
   --max-files 0                 \
   -e 0.01                       \
-  --hide filenames,dirnames,mouse,progress     \
+  --hide filenames,mouse,progress     \
```

Comment:
What does this do?

Reply:
Various gods I wondered where that script went to...